speech.yaml

```diff
@@ -226,32 +238,75 @@ components:
             - No
         first_name:
           type: string
-          description: The first name of the customer
+          description: The first name of the customer.
           maxLength: 10240
         last_name:
           type: string
-          description: The last name of the customer
+          description: The last name of the customer.
           maxLength: 10240
         email:
           type: string
           format: email
-          description: The email address of the customer
+          description: The email address of the customer.
         phone_number:
           type: string
           description:
             The phone number of the customer, typically from
-            ANI (Automatic Number Identification)
+            ANI (Automatic Number Identification).
           maxLength: 10240
         connection_id:
+          deprecated: true
           type: string
-          description:  The unique identifier of a specific connection profile
+          description:
+            The unique identifier of a specific connection profile.  This value
+            does not typically need to be set.  Please consult with your Medallia
+            Speech specialist to determine if a value is required.  This field has
+            been deprecated since Medallia Experience Cloud version 22CR3.0, please
+            use `connector_id` instead.
           maxLength: 10240
         profile_uuid:
+          deprecated: true
           type: string
           description:
-            The unique identifier (UUID) of the customer profile
-            (will be deprecated soon, use `connection_id` instead)
+            This field has been deprecated since Medallia Experience Cloud
+            version 21CR3.0, please use `connection_id` instead.
           maxLength: 10240
+        engine:
+          type: string
+          description:
+            The Medallia Speech engine to use to perform the
```

Comment:
Maybe here its worth mentioning that if you define an engine, there will be a locale+engine combo validation in the API.

Something like this but better worded: "If using engine parameter, please validate with your Medallia speech specialist which locales are supported for each engine" or "ask for recommended engine + locale combos"

Reply:
Does the existing line at the end of the description cover?

> The value to use is determined by your Medallia Speech specialist during the initial audio analysis process while onboarding to Medallia Speech.

Reply:
Yes, pretty much. 
Just wanted to be clear that a request like this:

locale: el  (greek)

can be accepted but if you also add an engine (which doesn't support that locale)

locale: el
engine: Engine 1

will be rejected.